### DISCUSSION TITLE
讨论组关了吧

### DISCUSSION THREAD

**Discussion by xiejun946** [WeNeedHome/SummaryOfLoanSuspension#1129] [General]
无聊，大陆的烂尾楼项目，台独分子和一些反动分子跑出来了？？？和你们有什么关系，跑这里来骂娘

**Comment by JayChou55** [comment]
沙比 讨论存在的问题就是 台独 没脑子的玩意。。。zf比你亲爹还亲，脑子是好东西 你却没有

**Comment by TALK1949** [comment]
五毛粉红从不关心中国人，却很在意关心中国苦难的人身份是不是敌特分子。😂
咱们来看看毛主席怎么说
![image](https://user-images.githubusercontent.com/39082705/184656761-7a1f75d8-b1f6-4611-b1b7-55a02fa2e509.png)

**Reply by 505Games-oss** [reply]
根据一些讲时政博主评论，这时候的毛是为了钓出反对他的人所使用的一记

**Reply by CKwuliuqi** [reply]
> 根据一些讲时政博主评论，这时候的毛是为了钓出反对他的人所使用的一记

呵呵，他老人家，连他的位置都不在乎，你们越是黑他，越证明他的伟大与正确

**Comment by oceanzhang860411** [comment]
> 无聊，大陆的烂尾楼项目，台独分子和一些反动分子跑出来了？？？和你们有什么关系，跑这里来骂娘

“计算机信息网络直接进行国际联网，必须使用邮电部国家公用电信网提供的国际出入口信道。任何单位和个人不得自行建立或者使用其他信道进行国际联网。
                ----《中华人民共和国计算机信息网络国际联网管理暂行规定》

你的主子有允許你出來嗎？

**Comment by Ventery** [comment]
别跟这帮垃圾废物一般见识，在墙内屁都不敢放一个。在这里可牛逼了还跟反动分子一起搅屎

**Reply by oceanzhang860411** [reply]
果然是當奴隸當習慣了

**Reply by MAGAWOLA** [reply]
独裁政权要是存在一丁丁的司法独立，还需要上百万受尽折磨的烂尾楼业主跑到墙外github联合起来维权？共产党只是中国几千年来因为畸形意识形态产生的一个三流政党，根据现在共产党需要高压统治维持政权，如果大家齐心协力进行非暴力抗争，这也是最有效最安全的方式，土共的独裁政权就会加速崩塌瓦解。届时自然会有新的民族领袖领导中国人民继续前进。粉红的下场也会跟当年的大清义和团一样，结局只有被主子叛卖和屠宰。

**Reply by TALK1949** [reply]
咋了？没有了删帖禁言因言获罪加持的墙外舆论环境让你感觉自卑了？

**Comment by Cegazhos** [comment]
台湾脑残是这样的 闻着味就来了

**Reply by MAGAWOLA** [reply]
粉红还是回到墙内接收共产党的颅内监禁吧，墙外没有粉红的生存土壤，别引战了

**Reply by Cegazhos** [reply]
> 粉红还是回到墙内接收共产党的颅内监禁吧，墙外没有粉红的生存土壤，别引战了

对对对，我是粉红，比你妈的批都粉

**Reply by tanuk1997** [reply]
> > 粉红还是回到墙内接收共产党的颅内监禁吧，墙外没有粉红的生存土壤，别引战了
> 
> 对对对，我是粉红，比你妈的批都粉

你妈批是黑的，别洗了
